src/main/java/seedu/masslinkers/logic/commands/ModAddCommand.java

```diff
@@ -20,6 +20,7 @@
 public class ModAddCommand extends ModCommand {
```

Comment:
Do you think it's a bug when we do `mod find 32123212`, it accepts this?
Ie. we did not make a strict requirement that mod find can only search for mods (this requirement was enforced for mod add)

Reply:
hmm is this considered a bug? can we change it during the feature freeze?